Document the new ID column. (#1600)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/dev/execution-plans.asciidoc

These operators are called _eager_ operators, and are denoted as such in <<execution-plan-operators-summary>>.
Eagerness can cause high memory usage and may therefore be the cause of query performance issues.

[[id-column]]
.IDs
Each operator is assigned a unique ID, which is shown in the execution plan.

> The IDs help referring unambiguously to operators.
> There are no guarantees about the order of IDs, although they will usually start with 0 at the root and increase toward the leaves of the tree.

Reviewer: This seems a little vague:
> There are no guarantees about the order of IDs, although they will usually start with 0 at the root and increase toward the leaves of the tree.

```suggestion
The IDs can be used to refer unambiguously to operators.
There are no guarantees about the order of IDs, although they will usually start with 0 at the root, and will increase towards the leaves of the tree.
```
